files/default/sql/ddl/3.3.0__initial_tables.sql

```diff
@@ -2282,10 +2282,10 @@ CREATE TABLE IF NOT EXISTS `hdfs_command_execution` (
 CREATE TABLE IF NOT EXISTS `monitoring_window_config` (
     `id` INT(11) NOT NULL AUTO_INCREMENT,
     `window_config_type` INT(11) NOT NULL,
-    `specific_id` INT(11),
+    `training_dataset_id` INT(11),
     `time_offset` VARCHAR(63),
     `window_length` VARCHAR(63),
-    `row_percentage` INT(11),
+    `row_percentage` FLOAT,
```

Comment:
How the row percentage will be represented with float? Is it from 0 to 1 for percentage and >1 number of rows?

Reply:
from 0 to 1, it is made clear in the python client